marketplace.xml

```diff
@@ -13163,6 +13159,8 @@ For more information <a target='NO_BLANK' href='http://www.datafor.com.cn/'>plea
       <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case1.jpg</screenshot>
       <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case2.jpg</screenshot>
       <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case3.jpg</screenshot>
+      <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case4.jpg</screenshot>
+      <screenshot>https://github.com/datafor123/visualizer-free/raw/master/resource/case5.jpg</screenshot>
```

Comment:
Unfortunately, only three screenshots are supported.

Reply:
Thanks for checking it out! We've made the changes and submitted it again.